Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/EngineApiWrapper.cs

```diff
@@ -91,6 +96,17 @@ public ScriptableByteArray CrdtGetState()
             }
         }
 
+        [PublicAPI("Used by StreamingAssets/Js/Modules/EthereumController.js")]
+        public object SendEthereumMessageAsync(int id, string method, string jsonParams)
```

Comment:
I believe JS `number` is `double`, not `int`. That's why you might get types mismatch